src/dashboard-sidebar-view.js

```diff
@@ -49,13 +49,14 @@ module.exports = cdb.core.View.extend({
 
     this._widgets.bind('add', this._maybeRenderWidgetView, this);
     this._widgets.bind('reset', this.render, this);
+    this._widgets.bind('orderChanged', this.render, this);
```

Comment:
Yes, we need it because `sort` event is triggered when a new item is added to the collection. We can't render the list each time a new widget is added in the process :S.